cfme/base/__init__.py

```diff
@@ -65,25 +85,37 @@ def collect_logs(self):
 
     @property
     def zone(self):
-        server_res = self.appliance.rest_api.collections.servers.find_by(id=self.sid)
-        server, = server_res
+        server, = self.appliance.rest_api.collections.servers.find_by(id=self.sid)
```

Comment:
Q: can you use `self.rest_api_entity` here?

Reply:
Will check it out